### PR TITLE
Updating with latest keywords

### PR DESCRIPTION
**Pull request by Ghoulboy78**

### CODE REVIEW

**Comment by ImUrX** [comment]
for some reason the last commit went back to the past

**Comment by ImUrX** [comment]
and why did you remove the print function?

File: src/completionProvider.ts [hunk 1]
```diff
@@ -363,16 +390,18 @@ export class ScarpetSet extends Set<vscode.CompletionItem> {
...
+      new vscode.CompletionItem("title", typesEnum.Function),
+      new vscode.CompletionItem("upper", typesEnum.Function),
       new vscode.CompletionItem("particle", typesEnum.Function),
       new vscode.CompletionItem("particle_line", typesEnum.Function),
       new vscode.CompletionItem("particle_rect", typesEnum.Function),
       new vscode.CompletionItem("create_marker", typesEnum.Function),
       new vscode.CompletionItem("remove_all_marker", typesEnum.Function),
       new vscode.CompletionItem("nbt", typesEnum.Function),
       new vscode.CompletionItem("escape_nbt", typesEnum.Function),
-      new vscode.CompletionItem("print", typesEnum.Function),
```
Comment (ImUrX) [inline src/completionProvider.ts:375]: why

File: package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
   "bugs": {
     "url": "https://github.com/ImUrX/vscode-scarpet/issues"
   },
-  "version": "0.1.8",
+  "version": "0.1.6",
```
Comment (ImUrX) [inline package.json]: why is this in the past, i recommend force-pushing to the second commit you did

**Review by ImUrX — CHANGES_REQUESTED** [review]

File: src/completionProvider.ts (hunk 1 above)
Comment (Ghoulboy78) [reply src/completionProvider.ts:375]: Cos it was already there above

File: package.json (hunk 2 above)
Comment (Ghoulboy78) [reply package.json]: I thought I had changed that...

Comment (Ghoulboy78) [reply package.json]: done

File: README.md [hunk 3]
```diff
@@ -1,17 +1,10 @@
 # scarpet for VSCode
 
-[![vscode version](https://img.shields.io/visual-studio-marketplace/v/ImUrX.scarpet)](https://marketplace.visualstudio.com/items?itemName=ImUrX.scarpet)
+## How to use
```
Comment (ImUrX) [inline README.md]: you just deleted what i did and copy+pasted it but dw, do it with this too

**Review by ImUrX — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Ghoulboy78) [reply README.md]: ah, I thought that was an accident

Comment (Ghoulboy78) [reply README.md]: but wait, what do u want me to do?

Comment (ImUrX) [reply README.md]: originally i wanted you to use git to do undo the commits that didnt update the code but now everything is a mess lol

Comment (Ghoulboy78) [reply README.md]: done

**Pull request merged**